scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/ram/RamCompute.java

```diff
@@ -0,0 +1,39 @@
+package org.ow2.proactive.scheduler.policy.ram;
+
+import java.io.Serializable;
+import java.lang.management.ManagementFactory;
+
+import org.objectweb.proactive.core.runtime.LocalNode;
+import org.objectweb.proactive.core.runtime.ProActiveRuntimeImpl;
+
+import com.sun.management.OperatingSystemMXBean;
+
+
```

Comment:
In the javadoc, it must be explained that it works only for SingleJVM deployments on a given machine